pinball/ui/data_builder.py

```diff
@@ -395,11 +396,17 @@ def _workflow_data_from_instances_data(instances):
                 if instance.start_time > start_time:
                     start_time = instance.start_time
                     last_instance = instance.instance
-            elif end_time is not None and instance.end_time > end_time:
-                start_time = instance.start_time
-                end_time = instance.end_time
-                status = instance.status
-                last_instance = instance.instance
+            elif end_time is not None:
```

Comment:
I forgot, what's the difference between instance.end_time == None and instance.end_time == sys.maxint?